components/cards/card.css

```diff
@@ -0,0 +1,241 @@
+}
+small{
+    color:var(--card-text);
+}
+/* 
+text on img
+*/
+.hero__c1{
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    height: 32rem;
+    width: 30rem;
+    margin: 4rem;
+    border: var(--card-text);
+    border-radius: 8px;
+    background-color: var(--card-bg-color);
+    overflow: hidden;
+    box-shadow: 2px 2px 6px 3px darkgrey;
+}
+.hero__text__overlay{
+    position: absolute;
+    margin-right: 16rem;
+    margin-top: 6rem;
+    font-weight: 500;
+    color: var(--card-text);
+}
+.img__c1{
+    position: relative;
+    height: 22rem;
+    width: 30rem;
+}
+/* 
+HORIZONTAL CARD/ WITH BADGE
+*/
+.card-img{
+    position: relative;
+    width: 14rem;
+    height: 17.5rem;
+    border-radius: 8px;
+}
+.badge{
+    position: absolute;
+    display: flex;
+    width: 14rem;
+    height: 1rem;
+    background-color: var(--card-badge);
+    top: 37.3%;
+    text-align: center;
+    color: var(--card-text);
+    justify-content: center;
+    align-items: center;
+}
+.btn{
+    
+    padding: 0.5rem ;
+    margin: 1.25rem 0;
+    background-color: transparent;
+    border: 2px solid grey;
+    font-size: 1rem;
+    text-align: center;
+    width: 12rem;
+    cursor: pointer;
+    font-weight: 600;
+}
+.btn-solid{
+    background-color: var(--card-btn);
+}
+.horizontal-card{
+    display: flex;
+    width: 100%;
+    background-color: var(--card-hr-bg);
+    color: var(--card-text);
+    align-items: center;
+    justify-content: center;
+    padding: 2rem 12rem;
+}
+
+.card-text{
+    padding: 3rem;
+    width: 33%;
+    display: flex;
+    flex-direction: column;
+    gap: 2rem;
+}
+.tag{
+    background-color: var(--card-tag);
+    color: var(--card-text);
+    padding: 3px 4px;
+    border-radius: 6px;
+}
+.price{
+    display: flex;
+    gap: 10px;
+    align-items: center;
+    justify-content: flex-start;
+}
+.line{
+    text-decoration:line-through;
+    opacity: 30%;
+}
+.bold{
+    font-weight: 600;
+}
+i{
+    margin: 8px;
+}
+
+/* 
+vertical card
+*/
+.card-end{
+    display: flex;
+    flex-direction: column;
+    gap: 8px;
+    justify-content: center;
+    align-items: center;
+}
+
+.card-vertical-box{
+    position: relative;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: var(--card-hr-bg);
+    gap: 8rem;
+}
+.vertical-card{
+    display: flex;
+    position: relative;
+    flex-direction: column;
+    gap: 1rem;
+    background-color: var(--card-hr-bg);
+    justify-content: center;
+    align-items: center;
+    color: var(--card-text);
+}
+.floating-icon{
+    position: absolute;
+    right: 1.5%;
+    top: 0%;
+    font-size: 26px;
+    color: var(--card-vr-floating-icon);
+}
+.bg{
+    background-color: var(--card-hr-bg);
+    border-radius: 50%;
+}
+.my{
+    margin-top: 8px;
+}
+.overlay{
+    opacity: 50%
+
+}
+.text-overlay{
+    position: absolute;
+    top: 37%;
+    font-size: 24px;
```

Comment:
Use of rems over px is preferred.so this is where you can make the appropriate changes.